exercises/run_stop_run/problem.fr.md

```diff
@@ -0,0 +1,65 @@
+Introduction aux générateurs ES6.
+
+## Infos
+
+Les générateurs sont des fonctions qui peuvent être interrompues
+en cours de route, et exécutées à nouveau plus tard, autant de
+fois qu’on le souhaite.  Par exemple :
+
+```js
+function *foo () {
+  var stop = yield 'woo';
+  console.log(stop, 'dans foo')
+}
+var G = foo();
+
+console.log( G.next() );
+// { value: 'woo', done: false }
+
+console.log( G.next('bar') );
+// 'bar', 'dans foo'
+// { value: undefined, done: true }
+```
+
+Il s’est passé quoi, là ?!  Il faut remarquer certains points :
+
+**1)** l’astérisque (`*`) — juste un indice que `foo` est un générateur.
+Vous pouvez le placer n’importe où entre le mot-clé `function` et le nom
+de la fonction, avec ou sans espaces autour.
+
+**2)** l’expression `yield` — elle suspend le générateur et renvoie `"woo"`.
+Ceci met le générateur en pause jusqu'à ce qu’on appelle sa méthode `next()`
+(voir point 3 ci-dessous).  On relancera le générateur en envoyant la chaîne
+`"bar"` via `next()`, qui deviendra du coup la valeur retournée par
+l’expression `yield` à l’intérieur de `foo()`, et sera donc affectée à la
+variable `stop`.  Les contextes sont sauvegardés à chque changement d’état
+« départ -> arrêt -> reprise ».
+
+**3)** Le résutlat de la fonction `foo()` est équipé d’une méthode `next()`,
+laquelle renvoie un objet de type `{ value: VALEUR_DU_YIELD, done: ON_A_FINI }`.
```

Comment:
Arg! It sounds so wrong when you know JS (the english version will need some fix as well)

"**3)** Le résutlat de la fonction `foo()` est équipé d’une méthode `next()`, laquelle renvoie un objet de type `{ value: VALEUR_DU_YIELD, done: ON_A_FINI }`." 

->

"**3)** Le résultat de la fonction `foo()` est un objet de type `{ value: VALEUR_DU_YIELD, done: ON_A_FINI }` qui dispose également d’une méthode `next()`."